flexget/utils/archive.py

```diff
@@ -209,7 +209,6 @@ def is_archive(path):
             archive.close()
             return True
     except (IOError, ArchiveError) as error:
-        error_message = 'Failed to open file as archive: %s (%s)' % (path, error)
-        log.debug(error_message)
+        log.debug('Failed to open file as archive: %s (%s)' % (path, error))
```

Comment:
you kinda missed the point. instead of interpolating the string yourself via `%` and then passing it to the logger, let it do the interpolation itself:
`log.debug('Failed to open file as archive: %s (%s)', path, error)`